BE/teamgu/src/main/resources/application.yml

```
@@ -12,11 +12,15 @@ spring:
       hibernate:
         dialect: org.hibernate.dialect.MySQL57Dialect
   datasource:
-    driver-class-name: com.mysql.cj.jdbc.Driver    
-    url: jdbc:mysql://teamgu-mysql-db.c9wk2jscggn1.ap-northeast-2.rds.amazonaws.com:3306/teamgu?useUnicode=true&characterEncoding=utf8&serverTimezone=Asia/Seoul&zeroDateTimeBehavior=convertToNull&rewriteBatchedStatements=true
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/teamgu?useUnicode=true&characterEncoding=utf8&serverTimezone=Asia/Seoul&zeroDateTimeBehavior=convertToNull&rewriteBatchedStatements=true
     hikari:
       username: root
-      password: teamgupassword
+      password: 1234
+#    url: jdbc:mysql://teamgu-mysql-db.c9wk2jscggn1.ap-northeast-2.rds.amazonaws.com:3306/teamgu?useUnicode=true&characterEncoding=utf8&serverTimezone=Asia/Seoul&zeroDateTimeBehavior=convertToNull&rewriteBatchedStatements=true
+#    hikari:
+#      username: root
+#      password: teamgupassword
```

```suggestion
    url: jdbc:mysql://teamgu-mysql-db.c9wk2jscggn1.ap-northeast-2.rds.amazonaws.com:3306/teamgu?useUnicode=true&characterEncoding=utf8&serverTimezone=Asia/Seoul&zeroDateTimeBehavior=convertToNull&rewriteBatchedStatements=true
      username: root
      password: teamgupassword
```

`Merge`가되면 배포 서버에서도 동일한 코드로 작동하기 때문에 `localhost mysql`이 존재하지 않아 에러를 유발합니다!
그래서 이 부분은 AWS RDS인 `teamgu-mysql-db`로 다시 바꿔주시면 됩니다~!

---

앗 바꿔서 올린다는걸 깜빡했네요 ㅎㅎ...😂 코드리뷰 감사합니다~!!